index.rst

```diff
@@ -7,652 +7,415 @@ Table of contents
-^^^^^^^^^
-
+   :caption: How-to guides
+
+   user/how-to-guides/how-to-organize-your-qubes
+   user/how-to-guides/how-to-update
+   user/how-to-guides/how-to-back-up-restore-and-migrate
+   user/how-to-guides/how-to-copy-and-paste-text
+   user/how-to-guides/how-to-copy-and-move-files
+   user/how-to-guides/how-to-copy-from-dom0
+   user/how-to-guides/how-to-install-software
+   user/how-to-guides/how-to-use-disposables
+   user/how-to-guides/how-to-enter-fullscreen-mode
+   user/how-to-guides/how-to-use-devices
+   user/how-to-guides/how-to-use-block-storage-devices
+   user/how-to-guides/how-to-use-usb-devices
+   user/how-to-guides/how-to-use-pci-devices
+   user/how-to-guides/how-to-use-optical-discs
+   user/how-to-guides/how-to-reinstall-a-template
+
+
+.. _templates:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Templates
 
-   Templates </user/templates/templates>
+   user/templates/templates
+   user/templates/fedora/fedora
+   user/templates/fedora/fedora-upgrade
+   user/templates/debian/debian
+   user/templates/debian/debian-upgrade
+   user/templates/minimal-templates
+   user/templates/xfce-templates
+   user/templates/windows/windows
 
-   Fedora templates </user/templates/fedora/fedora>
-
-   How to upgrade a Fedora template in-place </user/templates/fedora/fedora-upgrade>
-
-   Debian templates </user/templates/debian/debian>
-
-   How to upgrade a Debian template in-place </user/templates/debian/debian-upgrade>
-
-   Minimal templates </user/templates/minimal-templates>
-
-   Xfce templates </user/templates/xfce-templates>
-
-
-
-Troubleshooting
-^^^^^^^^^^^^^^^
 
+.. _troubleshooting:
 
 .. toctree::
    :maxdepth: 1
-
-   Installation troubleshooting </user/troubleshooting/installation-troubleshooting>
-
-   Update troubleshooting </user/troubleshooting/update-troubleshooting>
-
-   Updating Debian and Whonix </user/troubleshooting/debian-and-whonix-update-troubleshooting>
-
-   Hardware troubleshooting </user/troubleshooting/hardware-troubleshooting>
-
-   UEFI troubleshooting </user/troubleshooting/uefi-troubleshooting>
-
-   Autostart troubleshooting </user/troubleshooting/autostart-troubleshooting>
-
-   Suspend/resume troubleshooting </user/troubleshooting/resume-suspend-troubleshooting>
-
-   App menu shortcut troubleshooting </user/troubleshooting/app-menu-shortcut-troubleshooting>
-
-   Qube troubleshooting </user/troubleshooting/vm-troubleshooting>
-
-   HVM troubleshooting </user/troubleshooting/hvm-troubleshooting>
-
-   Disk troubleshooting </user/troubleshooting/disk-troubleshooting>
-
-   PCI troubleshooting </user/troubleshooting/pci-troubleshooting>
-
-   USB troubleshooting </user/troubleshooting/usb-troubleshooting>
-
-   GUI troubleshooting </user/troubleshooting/gui-troubleshooting>
-
-   Media troubleshooting </user/troubleshooting/media-troubleshooting>
-
-   VPN troubleshooting </user/troubleshooting/vpn-troubleshooting>
-
-
-
-Security in Qubes
-^^^^^^^^^^^^^^^^^
-
+   :caption: Troubleshooting
+
```

Comment:
QA: in the original index the order is as follows here under Security in Qubes section: 
[Multi-factor Login](https://www.qubes-os.org/doc/mfa/)
[CTAP proxy](https://www.qubes-os.org/doc/ctap-proxy/)